modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/api.ts

```diff
@@ -61,6 +61,8 @@ export async function getFields(fdsView: FDSViewType) {
 	const isObjectSchema =
 		responseJSON.components.schemas[restSchema].xml.name === 'ObjectEntry';
 
+	const localizablePropertySuffix = '_i18n';
```

Comment:
[nit] Constants should be on top of the file, with CAPS_LOCKED name. Alternatively, this can be inline.